app/assets/javascripts/select-kit/addon/components/selected-color.js

```diff
@@ -13,6 +13,7 @@ export default SelectedNameComponent.extend({
         el = document.querySelector(`[data-value="${color}"]`);
 
       if (el) {
+        el.style.borderBottom = "2px solid transparent";
         el.style.borderBottomColor = `#${color}`;
```

Comment:
```suggestion
        el.style.borderBottom = `2px solid #${color}`;
```

Reply:
feel free to do a follow up PR for cleanups Kris if you want.

Reply:
Cool cool cool 🙂